src/D2L.CodeStyle.Analyzers/Immutability/MutabilityInspector.cs

```diff
@@ -713,8 +445,164 @@ HashSet<ITypeSymbol> typeStack
 			}
 		}
 
-		private bool TypeIsFromOtherAssembly( ITypeSymbol type ) {
-			return type.ContainingAssembly != m_compilation.Assembly;
+		// Performs common type inspections, then splits off in to the various
+		// type-specific inspections that need to take place
+		private MutabilityInspectionResult PerformTypeInspection(

```

Comment:
Renamed and moved down from earlier in the file so that it's below all the "DoInspectFoo" methods.